sample-plugin/pom.xml

```
@@ -546,6 +546,13 @@
     </build>
     <profiles>
         <!-- Use .1 of most recent active line, and .3 (or later) of previous lines: https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/#choosing-a-version -->
+        <profile>
+            <id>2.361.x</id>
+            <properties>
+                <bom>2.361.x</bom>
+                <jenkins.version>2.361.1</jenkins.version>
```

```suggestion
                <jenkins.version>2.361</jenkins.version>
```
Should remain as weekly version, until 2.361.1 is released, otherwise the build fails.